godb/tasks/docker.yml

```diff
@@ -21,13 +21,34 @@
     repo: deb https://download.docker.com/linux/ubuntu focal stable
     state: present
 
+- name: debug
+  debug:
+    var: ansible_facts.service_mgr
+
 - name: install docker
   become: true
-  notify: restart docker
+  #notify: restart docker
   apt:
     name:
       - docker-ce
       - docker-ce-cli
       - containerd.io
+  register: apt_install
+
+- name: temporary fix for docker in docker
+  become: true
+  notify: restart docker
+  ansible.builtin.lineinfile:
+    path: /etc/init.d/docker
+    state: absent
+    regexp: 'ulimit -Hn 524288'
```

Comment:
What's the default systemd ulimit?

Reply:
Ignore that, just realised this is conditional on sysvinit so only for testing 😄. Can you add a comment?

Reply:
I think it's 524288.
Apparently making it a soft limit would also work (Removing the H flag)
I'm guessing making a smaller value would also work, but we aren't running sysvinit anywhere other than during molecule tests, so I'm not hugely concerned about removing it